scripts/get-workspace-publish-tag.js

```diff
@@ -0,0 +1,75 @@
+// Copyright 2021-2024 Buf Technologies, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import { readdirSync, readFileSync } from "fs";
+import { join } from "path";
+import { existsSync } from "node:fs";
+
+/*
+
+USAGE: node get-workspace-publish-tag.js
+
+Looks at the version used in the workspace, and returns one of:
+ - latest - for versions such as 3.1.4, 2.0.0, but also 0.1.2
+ - alpha - for versions such as 1.0.0-alpha.8
+ - beta - for versions such as 1.0.0-beta.8
+ - rc - for versions such as 1.0.0-rc.8
```

Comment:
This needs to be merged into v2, but then cutting an alpha will simply be `make setversion SER_VERSION=2.0.0-alpha.1 && make release`.

Reply:
Merged in https://github.com/bufbuild/protobuf-es/commit/9cf3b0bf8abe9f8abf15ffb9a18c9a1f632ba175